Sepolicy Addition for file sharing between guest OSes

taranpreetkaur0013 opened the pull request:
Sepolicy addition for file edit and share in VM guest OSes.
For multi-VM environment.

Tracked-On: OAM-89748
Signed-off-by: tkaur <taranpreet.kaur@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1634

taranpreetkaur0013 committed:
Sepolicy Addition for file sharing

Sepolicy addition for file edit and share in VM guest OSes.
For multi-VM environment.

Tracked-On: OAM-89748
Signed-off-by: tkaur <taranpreet.kaur@intel.com>

shyjumon-n reviewed (approved):
Looks good to me

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1560

lchen43 commented:
Why we put the policy in this file? Is there any relationship to health HAL?

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1563

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1564

Done by "Kaur, Taranpreet" <taranpreet.kaur@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/742

Done by "Kaur, Taranpreet" <taranpreet.kaur@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/1570

Done by "Kaur, Taranpreet" <taranpreet.kaur@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE: Trigger blocking tests Triggered: EAT(apl_nuc:celadon_ivi - userdebug), EAT(apl_nuc:r2_celadon_ivi - userdebug), EAT(kbl_nuc:celadon_ivi - userdebug), EAT(kbl_nuc:r2_celadon_ivi - userdebug)

More details, logs, and binaries: /absp/builders/celadon-premerge/builds/746

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Kaur, Taranpreet" <taranpreet.kaur@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/1449

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

Cactus automatically verified this PR.

For more information, see: /absp/builders/celadon-autobuild/builds/1645
